src/lib.rs

```diff
@@ -1,16 +1,12 @@
 //! The root library for blue-build.
-
+#![warn(clippy::correctness, clippy::suspicious, clippy::perf, clippy::style)]
 #![doc(
     html_logo_url = "https://gitlab.com/wunker-bunker/blue-build/-/raw/main/logos/BlueBuild-logo.png"
 )]
 #![doc = include_str!("../README.md")]
 #![forbid(unsafe_code)]
 #![allow(unused_imports)]
+#![allow(clippy::module_name_repetitions)]
```

Comment:
What does this end up doing?

Reply:
clippy throws warnings if the sturct or trait has the same ending as the file name.

ex: ContainerFileTemplate -> template.rs

I find this pretty annoying lol I like to defined an ending no matter the file

Reply:
Ok yeah makes sense